openfisca_core/variables.py

```diff
@@ -27,36 +27,42 @@
     bool: {
         'dtype': np.bool,
         'default': False,
-        'json_type': 'Boolean',
+        'json_type': 'boolean',
+        'formatted_value_type': 'Boolean',
```

Comment:
> The legislation explorer relies and these values, so I kept them. Swithing to something a little more standard would be probably better but this would introduce a breaking change, and would not add much value